.github/workflows/tests.yml

```diff
@@ -69,7 +70,7 @@ jobs:
         run: vendor/bin/phpunit --coverage-clover build/logs/clover-default.xml --coverage-filter ./src
 
       - name: Run tests using Relay
-        if: ${{ matrix.redis >= '6' }}
+        if: ${{ matrix.php != '8.4' && matrix.redis >= '6' }}
```

Comment:
Is this because there are no 8.4 builds for Relay?

Reply:
Yes, the tests were failing because the extension couldn’t be installed.